docs/actions/refresh-grid.md

```diff
@@ -0,0 +1,43 @@
+---
+id: refresh-grid
+title: Refresh Action Grid
+sidebar_label: Refresh Action Grid
+---
+
+> Audience: [`Citizen Developers`](/docs/audience#citizen-developers)<br/>
+> Skill Prerequisites: `Using Action Grid`
+
+This action is usually used to refresh the grid after executing another action.
+
+## `Typical Use Cases`
+
+- Refresh Action Grid with updated without reloading the entire page
+
+## `Actions Covered by this Document`
+
+| Action | Purpose |
+| -- | -- |
+| Refresh Action Grid | Refresh an Action Grid module without reloading the page |
+
+## `Input Parameter Reference`
+
+| Parameter | Description | Supports Tokens | Default | Required |
+| -- | -- | -- | -- | -- |
+| Select Module | Select the Grid Module you wish to refresh| Yes | Empty String | No |
+
+## `Javascript method`
```

Comment:
this is similar to the section in the Open Action Form in pop-up doc you reviewed last week where I suggested that perhaps we should leave it out and maybe link to it later when we have that documentation built. I am fine leave this in here because its clearly relevant, but just thought I would mention that from last week. Should we consider adding the API documentation back into the Open Action form in popup doc?

Reply:
I was going to ask you about that.  When I originally reviewed the first one it didn't seem like it needed to be there but the more I think about it the more I am leaning towards maybe we have it here as well.  Let's chat about some of these questions.  I can be available most of Monday if you have some time.